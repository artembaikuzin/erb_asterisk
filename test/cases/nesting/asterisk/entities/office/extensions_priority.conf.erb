<%= include_to 'extensions.conf.includes', priority: 999 %>

[hello-priority-inclusion]
<% content_for :outbound, priority: 333 do %>
exten => 1234,1,NoOp
<% end %>

<% apply_line_to :global_inbound, 'include => office-inbound-prior',
                 priority: 10 %>
