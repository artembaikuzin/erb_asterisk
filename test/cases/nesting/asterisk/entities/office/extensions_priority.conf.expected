; Included to "extensions.conf.includes"

[hello-priority-inclusion]



